health-checker/src/main/java/io/stargate/health/HealthCheckerActivator.java

```diff
@@ -16,9 +17,12 @@
 
   public static final String BUNDLES_CHECK_NAME = "bundles";
   public static final String DATASTORE_CHECK_NAME = "datastore";
+  public static final String STARGATE_NODES_CHECK_NAME = "stargate-nodes";
```

Comment:
Sorry, but I find the change in checker names confusing. The `datatore` checker used to validate that the `DataStore` in operational internally, i.e. queries can be made against data managed by current Stargate node itself. Now this is done by the `stargate-nodes` checks, but that name is too generic. Staragte nodes have many services by the checker validates only the internal code path of `DataStore` and `Persistence`... How about keeping the old `datastore` name for the checker that queries `system.local`?

As for the new checker that talks to storage nodes, how about naming it `storage`? WDYT?

Reply:
good idea, done